TypeScript/myPermCheck.ts

```diff
@@ -0,0 +1,59 @@
+// @ts-nocheck
+/**
+ * This is perm check, it allows users to parse the permission you want and let the plugin do the rest. (check bot or user for that perm).
+ *
+ * @author @NeoYaBoi [<@762918086349029386>]
+   @author @needhamgary [<@342314924804014081]
+ * @version 1.1.0
+ * @example
+ * ```ts
+ * import { permCheck } from "../plugins/myPermCheck";
+ * import { commandModule } from "@sern/handler";
+ * export default commandModule({
+ *  plugins: [ permCheck('bot or user', 'permission', 'No permission response') ],
+ *  execute: (ctx) => {
+ * 		//your code here
+ *  }
+ * })
+ * ```
+ */
+
+import { type GuildMember, PermissionResolvable } from "discord.js";
+import { CommandType, EventPlugin, PluginType } from "@sern/handler";
+export function permCheck(
+  who: string,
```

Comment:
restrict type to `user` and `bot`
also rename `who` to `target`

Reply:
made the change via default case.